src/public/bulkUpload.csv

```diff
@@ -0,0 +1,3 @@
+Add one proposal per row below. do NOT edit the template,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```

Comment:
Is this intended to make its way into the CSV that we deliver? I don't think we should include instruction text in the CSV.

Also, we had explicitly discussed allowing the template to be editable for power uploaders who want to add other base fields. Are you walking that back, or just postponing it?

Reply:
Happy to remove any type of instruction -- also FWIW I was ok with the idea that someone could add columns in an unsupported way.  The main issue is I don't want people thinking "ah, I'll just update this header to match MY field's label!"

Reply:
Yeah, I'd remove the instrux.